README.md

```diff
@@ -30,65 +30,80 @@ Development environment can help you debug problems in your application in the s
 The Perl container is set up so that Apache will load additionally provided .conf files located within the <code>cfg</code> directory of the application's root.  This is useful if you are configuring your application with a database backend and would want to pass through your environment variables to mod_perl with <code>PerlPassEnv</code>.
 
 ###Manual Installation: 
-1. Create an account at [https://www.openshift.com](https://www.openshift.com)  
-*NOTE*: OpenShift Online currently is using V2.
+1. Create an account at [https://www.openshiftapps.com](https://www.openshiftapps.com)  
 2. Fork a copy of [dancer-ex](https://github.com/openshift/dancer-ex)
 3. Clone your repository to your OpenShift server
 3. Update the GitHub repository url in the instant app configuration to match your forked url 
 2. Add a Perl application from the provided template
-`oc process -f openshift/templates/dancer.json | oc create -f - `
-3. Start the build  
-`oc start-build dancer-app`
+
+		$ oc process -f openshift/templates/dancer.json | oc create -f - 
```

Comment:
they need to override the source_uri during processing (-v arg) if they're going to use the forked repo.